docs/src/projections.md

```diff
@@ -61,6 +62,34 @@ ArchGDAL.createcoordtrans(source, target) do transform
 end
 ```
 
+## Reprojecting from a layer
+```@setup projections
+# Getting vector data
+ds = AG.read("/vsicurl/https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/metropole.geojson")
```

Comment:
Ideally we'd make a copy of it (licensing permitting) at https://github.com/yeesian/ArchGDALDatasets and fetch it from there so that this might not become a dead link in the future

Reply:
OK, I look into it.